base/exports.jl

```diff
@@ -981,6 +982,7 @@ export
     enumerate,
     next,
     start,
+    visit,
```

Comment:
I don't think we need to export both uppercase and lowercase any more. Let's stick with just one of them exported. Maybe only one of them defined at all.